python-backend/app/config.py

```diff
@@ -34,7 +34,12 @@ class Config(object):
     SQLALCHEMY_MAX_OVERFLOW = 20
     SQLALCHEMY_POOL_TIMEOUT = 300
 
-
+    #Flask-uploads configs
+    UPLOADED_DOCUMENT_DEST = '/app/document_uploads'
+    DOCUMENT_FILE_SET = set(['pdf', 'doc', 'docx'])
+    
+    #20MB file limit
+    FILE_BYTE_LIMIT = 20 * 1024 * 1024
```

Comment:
Is there a reason to save it as bytes? It looks like you save it as bytes here, just to convert it to MB later anyway